src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,30 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  modalInfo: Todo;
+  setModalInfo: (todo: Todo | null) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ modalInfo, setModalInfo }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    getUser(modalInfo.userId)
+      .then(setUser)
+      .finally(() => setIsLoading(true));
```

Comment:
```suggestion
     setIsLoading(true);
    
    getUser(modalInfo.userId)
      .then(setUser)
      .finally(() => setIsLoading(false));
```
You should set the loading state when you start loading something to show loader and remove it after finish